go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/stefannegrea/go-workers
```

Comment:
should this module reflect the `digitalocean/go-workers2` path?

Reply:
Indeed, I will fix it.

Reply:
Thank you for spotting the mistake.